slsim/FalsePositives/false_positive_pop.py

```diff
@@ -0,0 +1,116 @@
+from slsim.FalsePositives.false_positive import FalsePositive
+from typing import Optional
+from astropy.cosmology import Cosmology
+from slsim.Sources.source_pop_base import SourcePopBase
+from slsim.ParamDistributions.los_config import LOSConfig
+from slsim.Deflectors.deflectors_base import DeflectorsBase
+from slsim.Sources.source import Source
+from slsim.Deflectors.deflector import Deflector
+from slsim.lens_pop import draw_test_area
+import random
+
+
+class FalsePositivePop(object):
+    """Class to perform samples of false positive population."""
+
+    def __init__(
+        self,
+        elliptical_galaxy_population: DeflectorsBase,
+        blue_galaxy_population: SourcePopBase,
+        cosmo: Optional[Cosmology] = None,
+        los_config: Optional[LOSConfig] = None,
+        source_number_choice: Optional[list[int]] = [1, 2, 3],
+        weights_for_source_number: Optional[list[int]] = None,
+    ):
+        """
+        Args:
```

Comment:
formatting a bit odd. Also, please provide a better description of what the class does